resources/js/features/user-game-activity/components/UserGameActivityTimeline/SessionTimelineEvent/SessionTimelineEvent.tsx

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import type { FC } from 'react';
+
+import { formatDate } from '@/common/utils/l10n/formatDate';
+
+import { RichPresenceEventContent } from './RichPresenceEventContent';
+import { UnlockEventContent } from './UnlockEventContent';
+
+dayjs.extend(utc);
+
+interface SessionTimelineEventProps {
+  previousEventTimestamp: string | null;
+  previousEventKind: App.Enums.PlayerGameActivityEventType | 'start-session';
+  sessionEvent: App.Platform.Data.PlayerGameActivityEvent;
+}
+
+export const SessionTimelineEvent: FC<SessionTimelineEventProps> = ({
+  previousEventKind,
+  previousEventTimestamp,
+  sessionEvent,
+}) => {
+  const eventTime = formatDate(dayjs.utc(sessionEvent.when), 'LTS');
+
+  return (
+    <div className="flex flex-col gap-1 md:flex-row md:items-center md:gap-5">
```

Comment:
You probably have better UX philosophies than I do, but it feels like I'm scrolling a lot more than I'd like to. How much can this margin be decreased? 

Maybe I'm just used to the old layout, or I prefer to be able to get a better overview first.

Reply:
The timeline content is indeed taller, and I'm struggling to find an easy way to remedy that.

At the very least, I've managed to shave off ~4px in latest from each row, but that only makes any sort of tangible difference if there's a ton of events on the page.

I agree with your comment here https://github.com/RetroAchievements/RAWeb/pull/3006#discussion_r1900416946 regarding collapsible sections - it may mitigate some of the scrolling issue. This should probably come as a subsequent PR, as it will involve the creation of a new base component to support the functionality.